homeassistant/components/habitica/quality_scale.yaml

```diff
@@ -0,0 +1,80 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
```

Comment:
Is once per minute too aggressive for this API compared to the value you get? Just checking...

Reply:
the rate limit is 30 requests per 60s, and the rate limit also resets after 60s. The coordinator does 3 requests per refresh, leaving 27 requests for other operations. Should the rate limit be exceeded the integration will skip the update and try again after 60s. Service actions also handle this with an exception and a meaningful error message.

Reply:
Here is some more info about [rate limiting and rules for API calls](https://habitica.fandom.com/wiki/Guidance_for_Comrades#Rate_Limiting). Also the integration sends an X-Client header since recently, so the admins can identify requests and get in contact if there is a problem.

Reply:
Glad it's under the rate limit I was more asking from how the integration works from a user perspective and id new data is often available minutely. I'm not familiar with how this is used  and if having a poll that often is very useful or somewhat unnecessary?

Reply:
I already increased it from 30 to 60s after I took over maintenance of the integration 😬
On the other hand I didn't want to increase it too much. I had some complaints with the Bring integration that it takes too long (90s interval) from adding items to the list until the changes appear in Home Assistant. If it were only for the sensor data, I would agree with you, that it is unnecessary but users expect the to-do lists to be somewhat responsive.

Reply:
Thanks, yes, makes sense to me.  FWIW google tasks sets 30 minutes refresh, but maybe those users are less vocal. We could also consider adding a "refresh" button in the UI like calendar has and have a way to "poke" the update coordinator.